frontend/src/v5/ui/controls/dueDate/baseDueDate.component.tsx

```diff
@@ -38,17 +39,18 @@ export const BaseDueDate = ({ value: initialValue, disabled, onBlur, ...props }:
 		if (!open) setOpen(!disabled);
 	};
 	const onDateChange = (newValue) => {
+		const timestamp = !newValue ? null : newValue.toDate().getTime();
```

Comment:
I find the use of '!' here unusual, but I don't care too much

Reply:
Maybe thiw would be better?
```suggestion
		const timestamp = newValue?.toDate()?.getTime() || null;
```